.drone.nightly.yml

```diff
@@ -4,37 +4,37 @@ clone:
 
 build:
   integration-test:
-    image: $${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.30}
+    image: ${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.30}
```

Comment:
we were not intending too... does it not work anymore if we don't upgrade nightly?